app/src/layouts/adminLayout/adminLayout.jsx

```diff
@@ -1,30 +1,43 @@
-import React from 'react';
-import PropTypes from 'prop-types';
-import classNames from 'classnames/bind';
-import styles from './adminLayout.scss';
-import { AdminHeader } from './header';
-import { Sidebar } from './sidebar';
-
-const cx = classNames.bind(styles);
+/*
+ * Copyright 2018 EPAM Systems
+ *
+ *
+ * This file is part of EPAM Report Portal.
+ * https://github.com/reportportal/service-ui
+ *
+ * Report Portal is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Report Portal is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Report Portal.  If not, see <http://www.gnu.org/licenses/>.
+ */
 
-export const AdminLayout = ({ children }) => (
-  <div className={cx('admin-container')}>
-    <div className={cx('sidebar-container')}>
-      <Sidebar />
-    </div>
-    <div className={cx('content')}>
-      <div className={cx('header-container')}>
-        <AdminHeader />
-      </div>
-      <div className={cx('page-container')}>{children}</div>
-    </div>
-  </div>
-);
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import { Layout } from 'layouts/common/layout';
+import { AdminHeader } from './adminHeader';
+import { AdminSidebar } from './adminSidebar';
 
-AdminLayout.propTypes = {
-  children: PropTypes.node,
-};
+export class AdminLayout extends PureComponent {
```

Comment:
`PureComponent` is unnecessary here.
Let's leave it as a functional component.

Reply:
Done